index.html

```diff
@@ -17,28 +17,31 @@
         <div id="main">
             <div id="left">
                 <img src="pics/openageinverse.svg" alt="openage" id="logo">
-                <h1>A free (as in <i>freedom</i>) clone of the Age of Empires II engine</h1>
+                <h1>Free cross-platform RTS game engine based on Age of Empires mechanics</h1>
                 <div id="mainbuttons">
                     <a href="https://repo.openage.dev" class="code" id="button">View the Code</a>
-                    <a href="https://repology.org/project/openage/versions" class="tgz" id="button">Linux Packages</a>
                     <a href="https://bintray.com/simonsan/openage-packages/openage-windows-nightly/_latestVersion" class="tgz" id="button">Windows Packages</a>
+                    <a href="https://repology.org/project/openage/versions" class="tgz" id="button">Linux Packages</a>
                 </div>
             </div>
             <div id="rightcontainer"><div id="right">
-                <p>You know Age of Empires. You know its bugs. You know the fun playing it.</p>
-                <p>We are creating a free (as in freedom) clone of the Age of Empires II engine in C++17 from scratch (GPLv3+), using "modern" technologies like OpenGL, Vulkan, Python, Qt5, CMake, <a href="https://github.com/SFTtech/nyan/">nyan</a>, ...</p>
-                <p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively, minus features, plus additional epic bugs (or was it vice versa?).</p>
-                <p>Until we have free replacement assets, we can import original media files provided by the user.</p>
-                <p>openage is a huge and ambitious project, run by volunteers: Students and professionals from all over the world. We are glad about every helping hand, so the only requirement is your enthusiasm :)</p>
-                <p>Join us and help to create an awesome game!</p>
+				<p>We are creating a free (as in freedom, GPLv3+) cross-platform RTS game engine based on Age of Empires mechanics in C++17. We use "modern" technologies like OpenGL/GLSL, Python, Qt5, CMake, ...</p>
+				<p>This allows people using GNU/Linux, BSD, macOS or Windows to play the game natively.</p>
+				<p>We aim to create a game engine that provides the original look and feel. Until we have free replacement assets, we can convert original media files provided by the user to an format compatible to openage.</p>
```

Comment:
Would rephrase this because it is based on the previous mention of the "Age of Empires clone" whereas the scope is now much bigger. I would suggest merging these sentences into something like this:

> Our aim is to make openage a platform for the original Age of Empires games providing the same look and feel, but with more features for modding and multiplayer.